src/raylib/core/Window.hpp

```diff
@@ -70,6 +70,9 @@ namespace raylib
             /// @return The height of the window.
             static int getHeight();
 
+            /// @return Toggles fullscreen
+            static void toggleFullscreen();
```

Comment:
Why a **@return** ?

Reply:
It returns a full screen, you wouldn't understand...